src/test/scala/com/gu/configraun/models/ConfigurationTest.scala

```diff
@@ -6,20 +6,36 @@ import org.scalatest.{FreeSpec, Matchers}
 
 class ConfigurationTest extends FreeSpec with Matchers with Checkers with PropertyChecks {
 
-  "put config in, get config out" in {
-    Configuration.empty.withString("test1", "test2").getAsString("test1").right.toOption.get shouldBe "test2"
+  "put string config in, get string config out" in {
+    val conf = Configuration.empty.withString("test1", "test2").getAsString("test1")
+    conf.right.toOption.get shouldBe "test2"
+    conf.left.toOption shouldBe None
+  }
+
+  "put seq string config in, get seq string config out" in {
+    val conf = Configuration.empty.withList("test1", List("test2", "test3")).getAsList("test1")
+    conf.right.toOption.get shouldBe List("test2", "test3")
+    conf.left.toOption shouldBe None
   }
 
   "don't put config in, don't get config out" in {
-    Configuration.empty.getAsString("test1").left.get.message.startsWith(s"Parameter with key test1 is not present.") shouldBe true
+    val conf = Configuration.empty
+    conf.getAsString("test1").left.get.message.startsWith(s"Parameter with key test1 is not present.") shouldBe true
```

Comment:
Also, better to use scalatest matchers than to test startswith using a collection operation and then scalatest that the result is true. It'll be more readable and give you better error messages.

```scala
str should startWith ("prefix")
```
http://www.scalatest.org/user_guide/using_matchers#stringsAndArraysAsCollections